app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -497,6 +508,8 @@ class BrowserTabViewModel(
             canSharePage = true,
             showPrivacyGrade = true,
             canReportSite = true,
+            canWhitelist = canWhitelist,
```

Comment:
I wanted to check what `canWhitelist` means vs `isWhitelisted`. Now I see that in Home page that value is `false`. But if we visit any navigate to any website and we press back (navigating to home page again), the value remains as `true` and the option enabled. This is minor due to the check `val domain = site?.domain ?: return`.

Reply:
`canWhitelist` refers to whether the whitelisting feature is available similar to `canAddBookmarks` and `canShare`. `isWhitelisted` refers to if a site is actually whitelisted or not.

The issue you see was introduced globally at some recent point and will be fixed by https://app.asana.com/0/search/1175146161566013/1170812542831705. The menu state used to be reset and it no longer is which causes these quirks. But I can see that this is really icky and even more so when combined with this PR, so I'm going to pickup that issue as a priority. Thanks for flagging it!

Reply:
Thanks for the asana reference, I was not aware of this one.